### DISCUSSION TITLE
Could not resolve com.soywiz.korlibs.korim:korim:3.3.0

### DISCUSSION THREAD

**Discussion by aimdie** [korlibs/korge#1082] [Q&A] [answered]
Different versions of the library were downloaded, but there were build problems. I'm sure I'm not the only one with a problem.

 
:lib-ui:jvmTest: Could not resolve com.soywiz.korlibs.korim:korim:3.3.0.
Required by:
    project :lib-ui

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

**Comment by soywiz** [comment]
@aimdie can you provide a small project reproducing the problem so I can reproduce it? I never had that issue

**Comment by aimdie** [comment, marked as answer]
@soywiz 
I solved the problem，It needs to be added in "gradle.properties "：

kotlin.mpp.enableGranularSourceSetsMetadata=true

**Reply by soywiz** [reply]
Thanks! I have moved it to discussions so people can find the answer.
